tests/integration/Test_CLICommandsIntegration.php

```
@@ -0,0 +1,618 @@
+        $this->assertNotEmpty($ua, 'Profile should have user agent');
+
+        // Cleanup
+        unset($_SERVER['REMOTE_ADDR'], $_SERVER['HTTP_USER_AGENT']);
+    }
+
+    /**
+     * Test ResourceUri record creation.
+     */
+    public function test_resource_uri_creation()
+    {
+        $uri = '/test-page-' . time() . '/';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$uri'. (undefined)

(UnusedLocalVariable)